07-Customizing-AzSK-for-your-Org/Readme.md

```diff
@@ -971,7 +971,115 @@ Example: If you want to scan only the ARMControls present in your org-policy, th
 
 ![ARM Controls override](../Images/07_Custom_Policy_ARMControls.png)
 
-
 ##### Testing:
 
 Run clear session state command(Clear-AzSKSessionState) followed by services scan (Get-AzSKAzureServicesSecurityStatus). Scan should reflect configuration changes done.
+
+
+
+### How to increase/decrease attestation expiry period for the control with Org policy
+
+There are two methods with which attestation expiry period can be controlled using Org policy. 
+
+1. Update attestation expiry period for control severity 
+
+2. Update attestation expiry period for a particular control in SVT  
+
+**Note:** Expiry period can be customized only for statuses "WillNotFix", "WillFixLater" and "StateConfirmed". For status "NotAnIssue" and "NotApplicable", expiry period can be customized using "Default" period present as part of ControlSettings configuration.
+
+#### 1. Update attestation expiry period for control severity 
+   
+   Steps:
+
+   i) Go to control settings configuration present in module folder.  
```

Comment:
use ControlSettings instead of control settings as you have used it this way before